src/kind2Flow.ml

```diff
@@ -151,9 +153,13 @@ let status_of_trans_sys sys =
       | (n, Property.PropKTrue _) ->
         Format.eprintf "%s KU@." n;
         u+1,f
-      | (n, Property.PropFalse _) ->
+      | (n, Property.PropFalse _) when TSys.get_prop_kind sys n = Invariant ->
         Format.eprintf "%s FALSE@." n;
         u,f+1
+      | (n, Property.PropInvariant _) -> 
+        if TSys.get_prop_kind sys n = Invariant
+        then u,f
+        else (Format.eprintf "%s UNREACHABLE@." n; u,f+1)   
```

Comment:
You should update the description of the `status_of_trans_sys` function. Something like:
```
The exit status is
  * 0 if some properties are unknown or k-true (timeout),
  * 10 if some invariant properties are falsifiable (unsafe) or some reachability properties are unreachable,
  * 20 if all invariant properties are invariants (safe) and all reachability properties are reachable. *)
```